scripts/common_ci.py

```diff
@@ -135,8 +135,11 @@ def BuildMockICD(args):
     print("Run CMake for ICD")
     utils.make_dirs(ICD_BUILD_DIR)
     cmake_cmd = \
-        f'cmake -DCMAKE_BUILD_TYPE={args.configuration.capitalize()} -DBUILD_CUBE=NO -DBUILD_VULKANINFO=NO -DINSTALL_ICD=OFF -DVULKAN_HEADERS_INSTALL_DIR={EXTERNAL_DIR}/Vulkan-Headers/{BUILD_DIR_NAME}/install {args.cmake} ..'
-    RunShellCmd(cmake_cmd, ICD_BUILD_DIR)
+        f'cmake -DCMAKE_BUILD_TYPE={args.configuration.capitalize()} -DBUILD_CUBE=NO -DBUILD_VULKANINFO=NO -DINSTALL_ICD=OFF {args.cmake} ..'
+    cmake_prefix_path = os.environ.get('CMAKE_PREFIX_PATH', '') + ':' + os.path.join(VT_DIR, 'external', 'Vulkan-Headers', 'build', 'install')
+    cmake_prefix_env = os.environ.copy()
+    cmake_prefix_env['CMAKE_PREFIX_PATH'] = cmake_prefix_path
+    RunShellCmd(cmake_cmd, ICD_BUILD_DIR, env=cmake_prefix_env)
```

Comment:
I think its simpler to just use the helper that update_deps.py generates
```suggestion
        f'cmake -DCMAKE_BUILD_TYPE={args.configuration.capitalize()} -DBUILD_CUBE=NO -DBUILD_VULKANINFO=NO -DINSTALL_ICD=OFF -C {VT_DIR}/{EXTERNAL_DIR_NAME}/helper.cmake {args.cmake} ..'
    RunShellCmd(cmake_cmd, ICD_BUILD_DIR)
```

Reply:
:+1: yep, that's the way to go. Thanks!